scripts/feature_tests/upgrade/upgrade_common.sh

```diff
@@ -88,7 +88,7 @@ function deprovision_cluster() {
 
 function wait_for_cluster_deprovisioned() {
     echo "Waiting for cluster to be deprovisioned"
-    for i in {1..3600}; do
```

Comment:
this has been changed to 600 but later 1800, I am not understanding the logic/reason

Reply:
This is set to 600 because, cluster deprovisioning doesn't need longer time cycle.. For example controlplane provisioning needs longer time than this and that is the reason I use 1800 for 3 controlplane provisioning.